library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
We have the `vm_obj` `(#1 2 (#1 3 #0))` for the the `list nat` `[2, 3]`, and we have the `vm_obj` `[expr (list.cons (bit0 one) (list.cons (bit1 one) list.nil))` which is a pre-term `pexpr` that when elaborated will have type `list nat`.